integration/PetPortalServiceNowExtended/blueprint.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: xl/v1
+kind: Blueprint
+metadata:
+  projectName: Extended ServiceNow Pipeline
+  description: |
+    This is a more interesting release template that uses ServiceNow.
+  author: XebiaLabs
+  version: 1.0
+parameters:
+- name: FolderName
```

Comment:
same comment about folder as in the other blueprint, IMO this doesn't add any value

Reply:
I think you missed some of my comments about PetPortalServiceNowExtended blueprint, i'll add them again since the original link seems to be lost